Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/http_profile_model.ts

```diff
@@ -26,6 +26,10 @@ export class HttpProfileModel extends ProfileModel {
     return this.model.roles;
   }
 
+  public get groups(): Beam.DirectoryEntry[] {
+    return this.model.groups;

```

Comment:
Return a copy/slice. Doing so ensures that any modifications to the list do not affect the ```ProfileModel```.